Libraries/Opc.Ua.Server/Subscription/Subscription.cs

```diff
@@ -854,11 +862,11 @@ private NotificationMessage InnerPublish(
 
                     if ((monitoredItem.MonitoredItemType & MonitoredItemTypeMask.DataChange) != 0)
                     {
-                        ((IDataChangeMonitoredItem)monitoredItem).Publish(context, datachanges, datachangeDiagnostics);
+                        ((IDataChangeMonitoredItem)monitoredItem).Publish(context, datachanges, datachangeDiagnostics, m_maxNotificationsPerPublish);
                     }
                     else
                     {
-                        ((IEventMonitoredItem)monitoredItem).Publish(context, events);
+                        ((IEventMonitoredItem)monitoredItem).Publish(context, events, m_maxNotificationsPerPublish);
```

Comment:
Unsent messages are dropped in line 969, why don´t we just stop the message creation once there are enough? This way we discard all oldest messages. How do we handle this in a durable scenario. Do we really need a durable messageQueue?

Reply:
![image](https://github.com/user-attachments/assets/0e3d37c8-7cf6-462a-84c5-69c5ff1a2a45)